### DISCUSSION TITLE
Converting to one-page style？

### DISCUSSION THREAD

**Discussion by haotiansun14** [CaiJimmy/hugo-theme-stack#644] [Q&A] [answered]
Hi! Thank you so much for this great template. I was wondering if it is possible to put different pages into one page (Like the theme "academic")? It would be better looking since I am trying to make my personal website function as a resume. 
Thanks again for your help!

**Comment by CaiJimmy** [comment, marked as answer]
> I was wondering if it is possible to put different pages into one page (Like the theme "academic")?

Please elaborate. I haven't used Academic theme and don't know how it should be. 

But probably it's not supported by this theme since it's a blog theme, not a resume theme.

**Reply by haotiansun14** [reply]
Thanks, Jimmy! I'm trying to start by putting all pages (i.e. about, archives, search, links, etc.) together on the home page, would you mind giving some suggestions on which layout files should be the starting point? 

Tons of thanks!

**Reply by CaiJimmy** [reply]
The homepage is rendered by `layouts/index.html`. You should be able to copy and paste elements you need to it without big adjustments (maybe margins).

**Reply by haotiansun14** [reply]
Got it! Thank you so much!
